### DISCUSSION TITLE
How to activate Reminder to search for similar discussions

### DISCUSSION THREAD

**Discussion by vinckr** [community/community#20809] [Discussions] [answered]
Hello, in some repos I see this helpful 
- Reminder to [search for similar discussions](https://github.com/github-community/community/discussions?discussions_q=How%20to%20activate%20Reminder%20to%20search%20for%20similar%20discussions)
- or even a checkbox that says "I searched for similar discussions"

I could not find documentation on the feature, is it beta or how do I activate it for my repos? (e.g. [here](https://github.com/ory/kratos/discussions/new))

**Comment by daniddelrio** [comment]
Bumping this as there still seems to be no documentation on this. We would also like to show the "I have done a search for similar discussions" checkbox in our organization discussions.

**Comment by vinckr** [comment]
For reference, this is what it looks like
![Screenshot 2022-11-28 at 15 44 14](https://user-images.githubusercontent.com/55394283/212720971-2dbd859e-6225-450c-b22f-a8d10f9909c6.png)
see a reminder in action at [pinniped](https://github.com/vmware-tanzu/pinniped/discussions/new?category=general).

**Comment by daniddelrio** [comment, marked as answer]
Contacted Github Enterprise support and apparently this checkbox is not a feature to enable, but rather is shown when these conditions are met:
- The repo is public
- The repo has > 10 discussions
- The user does not have write access to the repo
- It's the first time the user is posting a discussion in the repo

Would've been nice if this was a feature that could be enabled 😅

**Reply by vinckr** [reply]
Oh that explains a lot, thanks for asking! 

I would love if I could enable this also for user who have opened discussions before, the other criteria seem reasonable to me.
